frontend/src/metabase-lib/v1/parameters/utils/filters.ts

```diff
@@ -26,7 +32,9 @@ export function fieldFilterForParameter(parameter) {
   return () => false;
 }
 
-export function columnFilterForParameter(parameter) {
+export function columnFilterForParameter(
```

Comment:
The only difference vs `master` is that this function accepts 3 arguments there.